sources/packages/web/src/views/aest/AssessmentsSummary.vue

```diff
@@ -32,9 +36,22 @@ export default {
       required: true,
     },
   },
-  setup() {
+  setup(props: any) {
+    const router = useRouter();
+    const goToStudentAppeal = (appealId: number) => {
+      router.push({
+        name: AESTRoutesConst.STUDENT_APPEAL_REQUESTS_APPROVAL,
+        params: {
+          studentId: props.studentId,
+          applicationId: props.applicationId,
+          appealId,
```

Comment:
Just a clarification for my understanding. studentId and applicationId are sent with key but appealId is just passed. So Vue automatically identifies the third property because that is only left one to be assigned. Is it?

Reply:
It is not Vue, it is typescript. When the property name and variable are the same we can omit the property name.
The above one would be equivalent to the below one:
```typescript
params: {
          studentId: props.studentId,
          applicationId: props.applicationId,
          appealId: appealId
}
```
We are doing this everywhere, for instance when we provide parameters to DB call, as below.
![image](https://user-images.githubusercontent.com/61259237/162061111-7a388ece-3482-4a5a-84b4-02aeda9b9897.png)
Thi one would be `{ userdId: userdId }`

Reply:
Got it. thanks for explaining 👍